### PR TITLE
Additional edits/fixes for Healpix

### PR DESCRIPTION
**Pull request by profxj**
As titled

### CODE REVIEW

**Review by tarraneh — CHANGES_REQUESTED** [review]
Few minor changes

File: astropath/healpix.py [hunk 1]
```diff
@@ -5,16 +5,22 @@
...
+from astropath import localization
+
+def elliptical_localization_to_healpix(
+    coord, PA, a, b, nside=None, 
+    resol=None, radius=None, fitsfile=None):
     """Convert an input localization ellipse around an input coordinate
     to a healpix Table
 
+    The input coordinates should be ICRS 
+
```
[the remark this answers is not in the document]
Comment (profxj) [reply astropath/healpix.py:23]: they can have any units.  But they must be an angle object.

File: astropath/healpix.py [hunk 2]
```diff
@@ -75,8 +81,38 @@ def elliptical_localization_to_healpix(coord, PA, a, b, nside=None,
...
+
+def localization_from_hpfile(hpix_file:str) -> dict:
+    """ Generate a localization dict from
+    a healpix file
+
+    Args:
+        hpix_file (str): 
+
+    Returns:
```
[the remark this answers is not in the document]
Comment (profxj) [reply astropath/healpix.py:99]: done

File: astropath/tests/test_healpix.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def test_ellipse():
     for key in ['UNIQ', 'PROBDENSITY']:
         assert key in hp_tbl.keys()
 
-    assert len(hp_tbl) == 4967
+    assert len(hp_tbl) == 4959
```
[the remark this answers is not in the document]
Comment (profxj) [reply astropath/tests/test_healpix.py:38]: added

**Review by tarraneh — APPROVED** [review]
Looks good

**Pull request merged**